response.cpp

```diff
@@ -230,18 +213,57 @@ std::string to_string(reply::status_type status)
 
 } // namespace stock_replies
 
-reply reply::stock_reply(reply::status_type status)
+void Response::SetStatus(const ResponseCode response_code)
+{
+	this->response_code = response_code;
+}
+
+void Response::AddHeader(const std::string& header_name, const std::string& header_value)
+{
+	header head;
+	head.name = header_name;
+	head.value = header_value;
+
+	headers.push_back(head);
+}
+
+void Response::SetBody(const std::string& body)
+{
+	content.append(body);
+}
+
+Response Response::stock_response(Response::ResponseCode rcode)
+{
+  std::string body = stock_responses::to_string(rcode);
+  Response resp;
+  resp.SetStatus(rcode);
+  resp.SetBody(body);
+  resp.AddHeader("Content-Length", std::to_string(body.size()));
+  resp.AddHeader("Content-Type", "text/html");
+  return resp;
+}
```

Comment:
umm yeah i guess that sounds good, or use the stock_response message directly but either way should work!